### DISCUSSION TITLE
Build multiple websites from one template?

### DISCUSSION THREAD

**Discussion by DivMode** [vercel/turborepo#1213] [Help] [answered]
Hey guys,

I was wondering how most people manage creating multiple websites/web apps from one template?

How would you manage a scenario where you build a website template (I’m using Next.js+sanity.io for headless cms) but want to reuse it for multiple projects?

So that if you want to make an edit to the base template it gets pulled into all the child projects automatically. Manually adding the same edits into multiple projects would be tedious and time-consuming so I was thinking there must be a better way.

After searching far I found turborepo but not sure if it fits what im looking for. Am I on the right track and monorepos are what I need?

**Comment by anthonyshew** [comment, marked as answer]
Shared packages! We recommend [Internal Packages](https://turbo.build/repo/docs/handbook/sharing-code/internal-packages).
